cmd/describe/machinepool/cmd.go

```diff
@@ -71,6 +71,7 @@ func DescribeMachinePoolRunner(userOptions DescribeMachinepoolUserOptions) rosa.
 			userOptions.machinepool = argv[0]
 		} else {
 			err := cmd.ParseFlags(argv)
```

Comment:
@hunterkepley I think we can drop this additional `cmd.ParseFlags` call as it should already be done by this point.

Reply:
@robpblake was able to drop that, but not the line mentioned in the first comment you made

Reply:
doesn't pass tests after removing